File: pyjl/tests_py2many/sys_argv.py
import sys
from typing import List


if __name__ == "__main__":
    # TODO: Use variable to help rust
    a: List[str] = sys.argv
    cmd: str = a[0]
    if cmd == "dart":
        pass
    else:
        assert "sys_argv" in cmd
    if len(a) > 1:
        print(a[1])
    else:
        print("OK")

File: tests/cases-py2many/sys_argv.jl


function main()
    a::Vector{String} = sys.argv
    cmd::String = a[1]
    if cmd == "dart"
        #= pass =#
    else
        @assert(findfirst("sys_argv", cmd) != Nothing)
    end
    if length(a) > 1
        println(a[2])
    else
        println("OK")
    end
end

main()
